modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -1032,6 +1032,26 @@ def test_insert(data):
     )
 
 
+def test_insert_4407():
+    data = {"col1": [1, 2, 3], "col2": [2, 3, 4]}
+    modin_df, pandas_df = pd.DataFrame(data), pandas.DataFrame(data)
+
+    def comparator(df1, df2):
+        assert_series_equal(df1.dtypes, df2.dtypes, check_index=False)
+        return df_equals(df1, df2)
+
+    for idx, value in enumerate((pandas_df.to_numpy(), np.array([[1]] * 3))):
```

Comment:
```suggestion
    for idx, value in enumerate((pandas_df.to_numpy(), np.array([[1]] * 3), np.array([[1, 2, 3], [4, 5, 6]])):
```
I think we should include the test thats supposed to fail to ensure both pandas and Modin fail correctly!